### PR TITLE
(feat): Remove progress bar

### PR DESCRIPTION
**Pull request by schristoff**
This could help with #2104

### CODE REVIEW

File: src/internal/packager/images/common.go [hunk 1]
```diff
@@ -107,9 +107,9 @@ func createPushOpts(cfg PushConfig, pb *message.ProgressBar) []crane.Option {
 	// TODO (@WSTARR) This is set to match the TLSHandshakeTimeout to potentially mitigate effects of https://github.com/zarf-dev/zarf/issues/1444
 	transport.ResponseHeaderTimeout = 10 * time.Second
 
-	transportWithProgressBar := helpers.NewTransport(transport, pb)
+	transportWithoutProgressBar := helpers.NewTransport(transport, nil)
```
Comment (AustinAbro321) [inline src/internal/packager/images/common.go:110]: The helpers.NewTransport is just [oras.NewTransport](https://github.com/oras-project/oras-go/blob/main/registry/remote/retry/client.go#L45) with a progressbar. Should we switch to that instead?

Comment (schristoff) [reply src/internal/packager/images/common.go:110]: I'm trying to keep the change minimal rn to see if it helps with the issue, however if this stays longterm we should do that

**Comment by AustinAbro321** [comment]
How would you feel about adding a message.info like we do in the new mirror? I know some users have many large images / have 100s of images. IMO we'll want to give some indication that the system is progressing
https://github.com/zarf-dev/zarf/blob/e1a99f67636ac40f8f2d1a260f5a4692bcfeebd8/src/internal/packager2/mirror.go#L126

**Comment by schristoff** [comment]
closing due to internal discussion

**Pull request closed without merging**